pkg/daemon/daemon.go

```diff
@@ -15,14 +15,13 @@ import (
 	ignv2_2types "github.com/coreos/ignition/config/v2_2/types"
 	"github.com/golang/glog"
 	drain "github.com/openshift/kubernetes-drain"
+	resr "github.com/openshift/machine-config-operator/lib/resourceread"
```

Comment:
Note: the rest of the code base uses `resourceread` without an alias, but I'm still OK with this.

Reply:
Fixed